aten/src/ATen/native/mps/operations/BinaryOps.mm

```diff
@@ -48,7 +85,42 @@ void binaryOpTensor(const Tensor& self, const Tensor& other, const Tensor& outpu
           newCachedGraph = new BinaryOpCachedGraph(mpsGraph);
           newCachedGraph->primaryTensor   = mpsGraphRankedPlaceHolder(mpsGraph, self_dtype , getMPSShape(self));
           newCachedGraph->secondaryTensor = mpsGraphRankedPlaceHolder(mpsGraph, other_dtype, getMPSShape(other));
-          newCachedGraph->outputTensor = binaryBlock(mpsGraph, newCachedGraph->primaryTensor, newCachedGraph->secondaryTensor);
+          MPSGraphTensor* primary = nil;
```

Comment:
Abhishek, can you please rebase your changes, because some of the code has changed.

Reply:
Done the rebase